lib/autoyast.pm

```diff
@@ -720,6 +721,27 @@ sub expand_variables {
     return $profile;
 }
 
+=head2 adjust_user_password
+
+ adjust_user_password($profile);
+
+ Password is defined at first, see lib/main_common.pm like below:
+ ---
+ $testapi::password = "xxxxxx";
+ $testapi::password = get_var("PASSWORD") if defined get_var("PASSWORD");
+ ---
+
+ $profile is the autoyast profile 'autoinst.xml'.
+
+=cut
+
+sub adjust_user_password {
+    my ($profile) = @_;
+    my $user_password = $testapi::password;
+    $profile =~ s/\{\{PASSWORD\}\}/$user_password/g;
```

Comment:
There's probably no use in making this an extra variable:
```suggestion
    $profile =~ s/\{\{PASSWORD\}\}/$testapi::password/g;
```

Reply:
Fixed